extensions/agroal/spi/src/main/java/io/quarkus/agroal/deployment/JdbcDriverBuildItem.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.agroal.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Register a JDBC driver for the Agroal extension.
+ * <p>
+ * It allows to resolve automatically the driver from the kind, thus users don't have to set the driver anymore, except if they
+ * want to use a specific one.
+ */
+public final class JdbcDriverBuildItem extends MultiBuildItem {
+
+    private final String kind;
+
+    private final String driverClass;
+
+    private final Optional<String> xaDriverClass;
+
+    public JdbcDriverBuildItem(String kind, String driverClass, String xaDriverClass) {
+        this.kind = kind;
```

Comment:
not mandatory but a future code reader will be helped by it being name dbKind

Reply:
Done.